api/src/main/java/org/openmrs/obs/ComplexData.java

```diff
@@ -36,11 +37,11 @@ public class ComplexData implements java.io.Serializable {
 	/**
 	 * Default constructor requires title and data.
 	 * 
-	 * @param title Name or brief description of ComplexData.
+	 * @param fILENAME Name or brief description of ComplexData.
 	 * @param data The complex data for an Obs
 	 */
-	public ComplexData(String title, Object data) {
-		setTitle(title);
+	public ComplexData(String fILENAME, Object data) {
```

Comment:
I'm not sure why this parameter needs to be renamed. ComplexData can store more than just files, though one use of this field is for filenames.